### PR TITLE
Feature/not found404

### PR DESCRIPTION
**Pull request by Mazurok001**

### CODE REVIEW

File: src/components/NotFound/NotFound.module.scss [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    padding-inline: 24px;
+  }
+
+  @include respond-to(desktop) {
+    padding-inline: 32px;
+  }
+}
+
+.image {
+  // outline: 1px solid orange ;
```
Comment (MaxBorovique) [inline src/components/NotFound/NotFound.module.scss:20]: remove all coments

File: src/components/NotFound/NotFound.module.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+  max-width: 500px;
+  width: 100%;
+  height: auto;
+  margin-bottom: 24px;
+}
+
+.title {
+  // outline: 1px solid orange ;
```
Comment (MaxBorovique) [inline src/components/NotFound/NotFound.module.scss:29]: same

File: src/components/NotFound/NotFound.module.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  
+  @include respond-to(tablet) {
+    font-size: 32px;
+    line-height: 1.3;
+    letter-spacing: -0.01em;
+  }
+}
+
+.description {
+  // outline: 1px solid orange ;
```
Comment (MaxBorovique) [inline src/components/NotFound/NotFound.module.scss:45]: same

File: src/components/NotFound/NotFound.module.scss [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  font-weight: 700;
+
+  @include respond-to(tablet) {
+    font-size: 20px;
+    letter-spacing: -0.01em;
+  }
+}
+
+.go_back {
+  // outline: 1px solid orange ;
```
Comment (MaxBorovique) [inline src/components/NotFound/NotFound.module.scss:60]: same

File: src/components/NotFound/NotFound.module.scss [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+  font-size: 16px;
+  line-height: 1.4;
+  color: $c-white;
+  margin-bottom: 32px;
+  font-weight: 700;
+
+  @include respond-to(tablet) {
+    font-size: 20px;
+    letter-spacing: -0.01em;
```
Comment (MaxBorovique) [inline src/components/NotFound/NotFound.module.scss:55]: sure u need this?

**Review by coldshein — APPROVED** [review]
Good job, comments with outline styles are not necessary

File: src/components/NotFound/NotFound.tsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+      <img
+        src={page_not_found}
+        alt="Page Not Found"
+        className={styles.image}
+      />
+      <h2 className={styles.title}>Oops! Page Not Found</h2>
+      <p className={styles.description}>
+        The page you&apos;re looking for doesn&apos;t exist or has been moved.
+      </p>
+      <a href="/" className={styles.go_back}>
```
Comment (MaxBorovique) [inline src/components/NotFound/NotFound.tsx:17]: change to <Link to={RoutesLink.HomePage}

**Review by MaxBorovique — CHANGES_REQUESTED** [review]

**Pull request merged**